app/src/main/AndroidManifest.xml

```diff
@@ -46,14 +47,23 @@
         <activity
             android:name="com.intervaltimer.google.activities.MainActivity"
             android:label="@string/app_name"
-            android:theme="@style/AppTheme.NoActionBar">
+            android:exported="true"
+            android:theme="@style/AppTheme.NoActionBar"
+            android:launchMode="singleTask">
+
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
 
-            <intent-filter android:autoVerify="true">
+            <intent-filter android:autoVerify="true"
+                tools:targetApi="m">
```

Comment:
What is this change doing?

Reply:
targetApi specifies for which version the element is available. https://developer.android.com/studio/write/tool-attributes#toolstargetapi
It was automatically recommended, since minSdk of this project is lower than the one necessary for intent-filters (21 vs 23).

Reply:
Did you consider to add this explanation to the corresponding commit message? It's sometimes tricky to understand why stuff was changed if you/someone else looks at this code in a few month...

Reply:
In this specific case, I believe that the explanation directly follows from what documentation says about targetApi element.